lib/Query/BuilderConverterSulu.php

```diff
@@ -0,0 +1,225 @@
+<?php
+
+namespace Sulu\Component\DocumentManager\Query;
+
+use PHPCR\Query\QOM\EquiJoinConditionInterface;
+use PHPCR\Query\QOM\QueryObjectModelFactoryInterface;
+use PHPCR\Query\QOM\QueryObjectModelConstantsInterface as QOMConstants;
+use Sulu\Component\DocumentManager\DocumentManager;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+use PHPCR\SessionInterface;
+use Doctrine\ODM\PHPCR\Query\Builder\AbstractNode as QBConstants;
+use Doctrine\ODM\PHPCR\Query\Builder\AbstractNode;
+use Doctrine\ODM\PHPCR\Query\Builder\BuilderConverterPhpcr;
+use Doctrine\DBAL\Query\QueryBuilder;
+use Doctrine\ODM\PHPCR\Query\Builder\SourceDocument;
+use Doctrine\ODM\PHPCR\Query\Builder\OperandDynamicField;
+use Sulu\Component\DocumentManager\MetadataFactoryInterface;
+
+/**
+ * Class which converts a Builder tree to a PHPCR Query
+ */
+class BuilderConverterSulu extends BuilderConverterPhpcr
```

Comment:
Should now extend the BuilderConverterBase class when (if) the relevanyt PHPCR-ODM PR is merged